### PR TITLE
task/3620 ssh_keys_choice page new design

### PR DESCRIPTION
**Pull request by tiwariav**

### CODE REVIEW

File: datacenterlight/views.py [hunk 1]
```diff
@@ -208,6 +209,7 @@ def validate_storage(self, value):
 
     @cache_control(no_cache=True, must_revalidate=True, no_store=True)
     def get(self, request, *args, **kwargs):
+
```
Comment (pcoder) [inline datacenterlight/views.py]: Not sure why we need this empty line and the following diff empty line

File: dynamicweb/settings/prod.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
 
+
```
Comment (pcoder) [inline dynamicweb/settings/prod.py]: Same here

File: hosting/forms.py [hunk 3]
```diff
@@ -20,9 +26,11 @@ def clean(self):
         password = self.cleaned_data.get('password')
         is_auth = authenticate(email=email, password=password)
         if not is_auth:
-            raise forms.ValidationError("Your username and/or password were incorrect.")
+            raise forms.ValidationError(
```
Comment (pcoder) [inline hosting/forms.py]: Can we add the translation function for the text
"Your username and/or password were incorrect." ?

File: datacenterlight/views.py (hunk 1 above)
Comment (tiwariav) [inline datacenterlight/views.py]: killed it.

File: dynamicweb/settings/prod.py (hunk 2 above)
Comment (tiwariav) [inline dynamicweb/settings/prod.py]: killed it too.

File: hosting/templates/hosting/base_short.html [hunk 4]
```diff
@@ -84,7 +85,7 @@
                           <ul id="g-account-menu" class="dropdown-menu" role="menu">
                             <li>
                                 <a href="{% url 'hosting:ssh_keys' %}">
-                                    <i class="fa fa-key"></i> {% trans "Keys"%}
+                                    <i class="fa fa-key"></i> SSH Keys
```
Comment (pcoder) [inline hosting/templates/hosting/base_short.html]: I think we should have a trans block here.
{% trans "SSH Keys"%}

File: hosting/templates/hosting/choice_ssh_keys.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+        window.location.href = '{{next_url}}';
+    </script>
+{% endif %}
+
+
+<script type="text/javascript">
+
+    window.onload = function () {
+        {% for user_key in keys %}
+            // var locale_date = new Date(document.getElementById("ssh-created_at-{{user_key.id}}").textContent).toISOString();
```
Comment (pcoder) [inline hosting/templates/hosting/choice_ssh_keys.html]: Could we remove this commented line?

File: hosting/templates/hosting/base_short.html (hunk 4 above)
Comment (tiwariav) [inline hosting/templates/hosting/base_short.html]: added

File: hosting/templates/hosting/choice_ssh_keys.html (hunk 5 above)
Comment (tiwariav) [inline hosting/templates/hosting/choice_ssh_keys.html]: removed.

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -18,359 +18,273 @@ msgstr ""
...
+msgid "Generate"
+msgstr "Generieren"
+
+msgid "Using existing key"
+msgstr "Nutzung eines existierenden SSH-Keys"
+
+msgid "I want to use my existing public key"
+msgstr "Ich möchte einen existierenden SSH-Key nutzen"
+
+#, fuzzy
```
Comment (pcoder) [inline hosting/locale/de/LC_MESSAGES/django.po]: I would remove the fuzzy, and possibly confirm if "Upload" really translates to "Schlüssel hochladen". I guess "Schlüssel hochladen" is "Upload Key"

File: hosting/static/hosting/css/bootstrap.min.css [hunk 7]
```diff
@@ -2,4 +2,4 @@
...
...nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs.nav-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border-bottom-color:#fff}}.nav-pills>li{float:left}.nav-pills>li>a{border-radius:4px}.nav-pills>li+li{margin-left:2px}.nav-pills>li.active>a,.nav-pills>li.active>a:focus,.nav-pills>li.active>a:hover{color:#fff;background-color:#777}.nav-stacked>li{float:none}.nav-stacked>li+li{margin-top:2px;margin-left:0}.nav-justified{width:100%}.nav-justified>li{float:none}.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-justified>li{display:table-cell;width:1%}.nav-justified>li>a{margin-bottom:0}}.nav-tabs-justified{border-bottom:0}.nav-tabs-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs-just
```
Comment (pcoder) [inline hosting/static/hosting/css/bootstrap.min.css]: I wonder whats changed in this file. If nothings changed, I would not make it a part of the PR.

File: hosting/locale/de/LC_MESSAGES/django.po (hunk 6 above)
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: Yes "Schlüssel hochladen" is "Upload Key"

**Pull request merged**